### ISSUE TITLE
darknet: ./src/utils.c:325: error: Assertion `0' failed.

### ISSUE THREAD

**Issue by JvitorS23** [pjreddie/darknet#2284]
I'm training a tiny yolov4 model and when the mAP will be calculated I get this error. Help, please.

**Comment by AnhPC03** [comment]
Could you upload full errors?

**Comment by JvitorS23** [comment]
> Could you upload full errors?

 (next mAP calculation at 1000 iterations) 
 999: 0.539764, 0.514114 avg loss, 0.002600 rate, 0.784127 seconds, 63936 images, 1.958418 hours left
Loaded: 0.000040 seconds
v3 (iou loss, Normalizer: (iou: 0.07, cls: 1.00) Region 30 Avg (IOU: 0.708076, GIOU: 0.697349), Class: 0.997288, Obj: 0.397894, No Obj: 0.002040, .5R: 0.937500, .75R: 0.479167, count: 48, class_loss = 0.461268, iou_loss = 0.092133, total_loss = 0.553400 
v3 (iou loss, Normalizer: (iou: 0.07, cls: 1.00) Region 37 Avg (IOU: 0.676004, GIOU: 0.650890), Class: 0.991853, Obj: 0.305354, No Obj: 0.000455, .5R: 0.888889, .75R: 0.288889, count: 45, class_loss = 0.486947, iou_loss = 0.202898, total_loss = 0.689844 
 total_bbox = 95331, rewritten_bbox = 0.020980 % 

 (next mAP calculation at 1000 iterations) 
 1000: 0.474444, 0.510147 avg loss, 0.002610 rate, 0.784307 seconds, 64000 images, 1.959921 hours left

 calculation mAP (mean average precision)...
 Detection layer: 30 - type = 28 
 Detection layer: 37 - type = 28 
4
 CUDA Error Prev: an illegal memory access was encountered
CUDA Error Prev: an illegal memory access was encountered: Success
darknet: ./src/utils.c:325: error: Assertion `0' failed.

**Comment by JvitorS23** [comment]
I changed my yolov4-tiny.cfg file and it works now. Thanks

**Comment by korhun** [comment]
@JvitorS23 I have the same error with yolov4-tiny.cfg; which I edited for 1 class. Which changes resolved this issue?

**Comment by JvitorS23** [comment]
I don't remember. I just downloaded other .cfg file.
